mne/viz/tests/test_raw.py

```diff
@@ -476,65 +465,66 @@ def test_plot_raw_traces(raw):
     with pytest.warns(None):  # sometimes projection
         raw.plot(group_by='position', order=np.arange(8))
     for fig_num in plt.get_fignums():
-        fig = plt.figure(fig_num)
-        if hasattr(fig, 'radio'):  # Get access to selection fig.
+        sel_fig = plt.figure(fig_num)
+        if hasattr(sel_fig, 'radio'):  # Get access to selection fig.
             break
     for key in ['down', 'up', 'escape']:
-        fig.canvas.key_press_event(key)
+        fig._fake_keypress(key, fig=sel_fig)
```

Comment:
you changed what is being tested here. it's now sending those fake keypresses to `fig` instead of `sel_fig`.  You should revert renaming `fig` to `sel_fig`.  I think also maybe we can get access to `sel_fig` through `fig.mne.fig_selection` instead of looping and looking for `radio` attribute?

Reply:
It should still be the same logic:
`sel_fig` is passed to the `_fake_keypress`-method so the `key_press_event` is called on `sel_fig`.
But the problem why I had to rename `fig` to `sel_fig` is that `_fake_keypress`-method only exists in `BrowserBase` which is inherited from `fig`(`MNEBrowseFigure`) and not `sel_fig`(`MNESelectionFigure`).
This was the cost of generalizing all backend-agnostic methods from the browser and interactivity-tests into one base-class, which can't be inherited by child figures as `MNESelectionFigure`.

Reply:
ah, you're right, I was misreading the code